docs/release-notes/8.7.asciidoc

```diff
@@ -10,6 +10,14 @@
 ==== Known issues
 * After alerts are generated for the first time, you may have to refresh your browser before your alert data appears on pages that use data views (for example, Timeline). Navigating between pages will not work (https://github.com/elastic/security-docs/issues/3046[#3046]).
 
+
+*{agent} upgrade from 8.6.2 to 8.7.2 can fail:*
```

Comment:
While this adds more visual emphasis, we prob don't need to doc this known issue in its own section. If we did end up creating a sep section for this, we'd need to also change the formatting for the [other known issue](https://github.com/elastic/security-docs/pull/3120/files#diff-aced6a718123f207068aaa452213d0c426a62eb87e6c511397ec7570fa03b0cbL11).
 
```suggestion

```

Reply:
in addition, this should say `{agent} upgrade from 8.6.2 to 8.7.0 can fail:`. There is a typo with `8.7.2`